README.md

```diff
@@ -25,6 +25,8 @@ For the management calls, the process needs to have the `CAP_NET_BIND_SERVICE` c
     * _Read._ Return the ASCII-encoded Signal Strength of the Unit in dBm
   * Get Wifi connection status characteristic `5c305a15-6859-4d6c-a87b-8d2c98c9f6f0`
     * _Read._ Return the ASCII-encoded SSID of current wifi connected to the unit
+  * Set Wifi Connection characteristic `5c305a16-6859-4d6c-a87b-8d2c98c9f6f0`
+    * _Write._ Send in wifi SSID and Password of wifi to connect to
```

Comment:
Once the API settles, might we describe what exactly one submits and in what order here?